hooks/playbooks/ironic_inject_tls.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Inject CA bundle in ironic-python-agent and create images
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  vars:
+    _namespace: openstack
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  tasks:
+    - name: Collect facts
+      ansible.builtin.setup:
+        gather_subset:
+          - '!all'
+          - '!min'
+          - user_gid
+          - user_id
+
+    - name: Create directory
+      ansible.builtin.file:
+        path: "{{ cifmw_basedir }}/artifacts/ironic_ipa_tls/initramfs"
+        state: directory
+        mode: "0755"
+
+    - name: Get internal-ca-bundle and IPA image from k8s resources
+      ansible.builtin.set_fact:
+        internal_ca_bundle: >-
+          {{
+            query(
+              'kubernetes.core.k8s',
+              api_version='v1',
+              kind='Secret',
+              resource_name='combined-ca-bundle',
+              namespace=_namespace
+            ) | first
+              | community.general.json_query('data."internal-ca-bundle.pem"')
+          }}
+        ironic_python_image: >-
+          {{
+            query(
+              'kubernetes.core.k8s',
+              api_version='v1beta1',
+              kind='Ironic',
+              resource_name='ironic',
+              namespace=_namespace
+            ) | first
+              | community.general.json_query('spec.images.ironicPythonAgent')
```

Comment:
ditto - if we can avoid `json_query`, it would be better

Reply:
For this case I changed it to use map instead. It works because there is no `.` in the key.